documentation/feature/medical-system.md

```diff
@@ -125,83 +144,101 @@ Advanced medical system brings more different types of wounds, each with its own
  
 #### 3.1.5 Bandage types
+
 Advanced medical system brings 4 different types of bandages and also properly working tourniquet. Below there is a list of all bandage types with effectiveness vs different wound types. Higher effectiveness bandages needs to be applied fewer times than ones with lower effectiveness. That depends on wound size. Applying bad type of bandage on given wound can yield wound opening after a while if not stiched fast enough.
 All bandage types weights about 50 grams each.
 
-* Bandage (basic)
+* Bandage (basic) is best for:
   * Abrasions - highest effectiveness
-  * Avulsions - lowest effectiveness
-  * Contusions - highest effectiveness
-  * Crush wounds - low effectiveness
-  * Cut wounds - very low effectiveness
   * Lacerations - medium effectiveness
-  * Velocity wounds - lowest effectiveness
-  * Puncture wounds - low effectiveness
 
-* Bandage (packing)
+* Bandage (packing) is best for
   * Abrasions - highest effectiveness
   * Avulsions - highest effectiveness
-  * Contusions - highest effectiveness
-  * Crush wounds - low effectiveness
-  * Cut wounds - lowest effectiveness
-  * Lacerations - lowest effectiveness
   * Velocity wounds - highest effectiveness
-  * Puncture wounds - lowest effectiveness
 
-* Bandage (elastic)
+* Bandage (elastic) is best for:
   * Abrasions - highest effectiveness
-  * Avulsions - lowest effectiveness
-  * Contusions - highest effectiveness
   * Crush wounds - highest effectiveness
   * Cut wounds - highest effectiveness
   * Lacerations - highest effectiveness
-  * Velocity wounds - low effectiveness
   * Puncture wounds - high effectiveness
 
-* QuikClot
-  * Abrasions - medium effectiveness
-  * Avulsions - high effectiveness
-  * Contusions - medium effectiveness
-  * Crush wounds - medium effectiveness
-  * Cut wounds - medium effectiveness
-  * Lacerations - medium effectiveness
-  * Velocity wounds - high effectiveness
-  * Puncture wounds - high effectiveness
+* QuikClot is best for: 
+  * All types of wounds - medium effectiveness
+  * Special: It has lowest wound opening chance. It is often needed to apply it more than once, depening on wound type and size.
 
 * Tourniquet
   * Can only be applied on limbs
   * Stops bleeding from wounds
+  * Should be used only when there are more than one wounds
   * Should be taken off as fast as possible and applied only to give medic time to bandage all the wounds
   * If not taken off for a while it will cause pain to patient, can cause death that way
 
 #### 3.1.6 Transfuzions
-In case of blood loss, blood could be replenished by three different types of IV: blood, plasma and saline. We distinguish 3 different sizes of IV bags: 250ml, 500ml and 1000ml.
+
+In case of blood loss, identified by lower blood pressure, blood could be replenished by three different types of IV: blood, plasma and saline. We distinguish 3 different sizes of IV bags: 250ml, 500ml and 1000ml.
```

Comment:
low blood pressure is not necessarily an indicator of blood loss. The system will tell you if a patient has lost a lot of blood however. BP and HR are indicators and should be interpreted as such. Alone they won't tell you the whole story.